src/pages/contact_information.tsx

```diff
@@ -0,0 +1,107 @@
+	title: string;
+	operationTitle: string;
+	operationTime: string;
+	imgUrl: string;
+	email: string;
+}
+
+const imageUrl = "/assets/images/logos/registrar.png";
+
+const contactCards = [
+	{
+		imgUrl: imageUrl,
+		title: "Academic Support",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "ug_academic@stamford.edu",
+	},
+	{
+		imgUrl: imageUrl,
+		title: "Registrar",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "registrar.bkk@stamford.edu",
+	},
+	{
+		imgUrl: imageUrl,
+		title: "Library",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "library@stamford.edu",
+	},
+	{
+		imgUrl: imageUrl,
+		title: "Finance",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "cashier_rm9@stamford.edu",
+	},
+	{
+		imgUrl: imageUrl,
+		title: "IT Support",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "support@stamford.edu",
+	},
+	{
+		imgUrl: imageUrl,
+		title: "Student Service",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "international-service@stamford.edu",
+	},
+];
+
+const ContactCard = ({ title, operationTime, operationTitle, imgUrl, email }: ContactCardProps) => (
+	<Card className="mx-auto max-w-[17rem] rounded-lg p-10 md:p-10 lg:p-10">
+		<CardSection className="max-w-xs">
+			<Image className="h-full w-full rounded-lg" src={imgUrl} alt="Contact Img" />
+		</CardSection>
+
+		<CardSection className="font-open-sans mt-4 flex text-lg font-semibold text-white md:mt-6 lg:mt-8">
+			{title}
+		</CardSection>
+
+		<CardSection className="mt-2 flex items-center text-sm md:mt-4 md:text-base lg:mt-6">
+			{operationTitle}
+			<br />
+			{operationTime}
+		</CardSection>
+
+		<Card.Section className="mt-4">
+			<div className="w-full">
+				<a href={`mailto:${email}`}>
+					<Button className="h-10 w-full bg-slate-800 text-blue-300">Email</Button>
+				</a>
+			</div>
+		</Card.Section>
+	</Card>
+);
+
+const ContactInfoPage = () => (
+	<div className="container mx-auto mt-12 justify-center">
+		{/* Header */}
+		<h1>
+			<div className="flex flex-col items-center justify-center text-center">
+				<div className="mb-4 text-5xl font-bold text-white">Contact Information</div>
+			</div>
+		</h1>
+
+		{/* Header.Description */}
+		<Text className="text-center">
+			Explore the diverse departments below to find the contact information you need. Our dedicated staff is ready
+			to help you on your educational journey. Feel free to reach out, and we'll ensure you get the support and
+			information required to succeed during your time at Stamford
+		</Text>
+
+		<Group className="m-8 grid items-center justify-center gap-8 sm:grid-cols-1 md:grid-cols-2 lg:grid-cols-3">
+			{contactCards.map((card) => (
+				<ContactCard key={`contactcard-${card.title}`} {...card} />
+			))}
+		</Group>
```

Comment:
- Use Grid component for grids tbh, it's quite cool.
- Try to use Mantine's way of styling when possible since some styles will otherwise conflict.
- Gap was too far apart, this adds some padding on the x axis along with changing the margin so that it only affects the y axis.